src/iterative_ensemble_smoother/_iterative_ensemble_smoother.py

```diff
@@ -28,16 +28,12 @@ def __init__(
         min_steplength: float = 0.3,
         dec_steplength: float = 2.5,
     ):
-        self._module_data = ModuleData(ensemble_size)
+        self.iteration_nr = 1
         self._ensemble_size = ensemble_size
```

Comment:
could maybe be renamed to `_initial_ensemble_size`

Reply:
Agree 👍